Codewars1.py

```diff
@@ -0,0 +1,67 @@
+''' In this kata you will create a function
+that takes a list of non-negative integers
+and strings and returns a new list with the
+strings filtered out.'''
+
+
+def filter_list(l):
+    i = 0
```

Comment:
please don't use magic names l, i

Reply:
l is for 'list', i is for iteration

Reply:
better to use names that show what do they keep